docs/pages/access-controls/guides/device-trust.mdx

```diff
@@ -63,9 +72,15 @@ layer of protection.
 
 (!docs/pages/includes/commercial-prereqs-tabs.mdx!)
 
-- A signed and notarized `tsh` binary for enrollment.
-  [Download the macOS tsh installer](../../installation.mdx#macos).
-- A macOS device to register and enroll.
+- To enroll a macOS device:
+  - A signed and notarized `tsh` binary for enrollment.
+    [Download the macOS tsh installer](../../installation.mdx#macos).
+  - A Teleport version newer than v12.0.0.
+- To enroll a Windows device:
+  - A device that includes a TPM with 2.0 support.
+  - `tsh` installed on the device. [Download the Windows tsh installer](../../installation.mdx#windows-tsh-client-only).
+  - Credentials for a user with administrator privileges for the device. This is only required during enrollment.
+  - A Teleport version newer than v13.1.2.
```

Comment:
Revisit this version number before merging.

Reply:
Would TPM support, by definition, bump the version to 13.2?

Reply:
It'll be going out in the next patch release, rather than waiting for a minor release as per conversations with @zmb3